lib/config.js

```diff
@@ -323,6 +323,16 @@ const calculateValue = () => {
         zodgame: {
             cookie: envs.ZODGAME_COOKIE,
         },
+        openai: {
+            openai_key: envs.OPENAI_KEY,
+            model: envs.OPENAI_MODEL || 'gpt-3.5-turbo-16k-0613',
```

Comment:
```suggestion
            model: envs.OPENAI_MODEL || 'gpt-3.5-turbo-16k',
```
It would be better to not use snapshot models since they will be removed after.